### ISSUE TITLE
Warnings when building with Xcode 8.3 (Swift 3.1)

### ISSUE THREAD

**Issue by vrutberg** [malcommac/SwiftDate#403]
When building a project that depends on SwiftDate, I get the following warnings from the SwiftDate module:

```
/Users/vrutberg/code/iphone-app/Pods/SwiftDate/Sources/SwiftDate/CalendarName.swift:156:6:
 'unsafeBitCast' from 'UnsafeMutablePointer<CFAbsoluteTime>' (aka 
'UnsafeMutablePointer<Double>') to 'UnsafeMutablePointer<CFAbsoluteTime>' (aka 
'UnsafeMutablePointer<Double>') is unnecessary and can be removed
```

```
/Users/vrutberg/code/iphone-app/Pods/SwiftDate/Sources/SwiftDate/CalendarName.swift:158:6: 
'unsafeBitCast' from 'UnsafeMutablePointer<CFTimeInterval>' (aka 
'UnsafeMutablePointer<Double>') to 'UnsafeMutablePointer<CFTimeInterval>' (aka 
'UnsafeMutablePointer<Double>') is unnecessary and can be removed
```

**Comment by sumowesley** [comment]
Same here.

**Comment by rogpfreck** [comment]
Same here too.

**Comment by frederik-jacques** [comment]
Same here

**Comment by malcommac** [comment]
Fixed in #404 thanks to @frederik-jacques.
I'll publish 4.0.14 later today.

**Comment by vrutberg** [comment]
Great!
